modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -1576,6 +1579,32 @@ <h4>Color Functions <span class="grammar" style="font-size: smaller;">&lt;color-
           href="#typecolor"
           class="typelink">&lt;color&gt;</a>
         ) ]+ </span></p>
+    <h3><a id="typetextbounds">&lt;text-bounds&gt;</a></h3>
+    <p>The geometry of text can be measured either in terms of the bounds of
+       the particular text to be rendered - visual bounds, or as properties
+       of the font and the characters to be rendered - logical bounds.
+       Visual bounds are more useful for positioning text as graphics, and
+       for obtaining tight enclosing bounds around the text.</p>
+    <p>Logical bounds are important for laying out text relative to other
+       text and other components, particularly those which also contain text.
+       The bounds aren't specific to the text being rendered, and so will
+       report heights which account for the potential ascent and descent of
+       text using the font at its specified size. Also leading and trailing
+       spaces are part of the logical advance width of the text.</p>
+    <p>&nbsp;</p>
+    <p class="grammar">[ logical | visual | logical-vertical-center ]</p>
+    <ul>
+      <li><strong>logical</strong>: The logical bounds are based on font metrics information.
+          The width is based on the glyph advances and the height on the ascent, descent, and
```

Comment:
Should this perhaps be "of" ?
```suggestion
          The width is based on the glyph advances and the height of the ascent, descent, and
```